Makefile

```diff
@@ -22,6 +22,10 @@ UI_REPO = git@github.com:IBM/FfDL-dashboard.git
 CLI_CMD = $(shell pwd)/cli/bin/ffdl-$(UNAME_SHORT)
 CLUSTER_NAME ?= mycluster
 PUBLIC_IP ?= 127.0.0.1
+TRAVIS_IMAGES_VERSION ?= 0.0.1-master
+TEST_IMAGES = $(addprefix $(DOCKER_NAMESPACE)/, $(TEST_IMAGES_SUBFIX))
```

Comment:
nitpick: I think "suffix" would be the more common terminology (`TEST_IMAGES_SUFFIX`)